packages/infield-button/src/spectrum-infield-button.css

```diff
@@ -11,144 +11,190 @@ governing permissions and limitations under the License.
+    --mod-infield-button-icon-color-key-focus: var(--mod-infield-button-icon-color-key-focus-disabled, var(--spectrum-disabled-content-color));
 }
 
-::slotted(*) {
-    color: var(--mod-infield-button-icon-color, var(--spectrum-infield-button-icon-color));
+:host([size='s']) {
+    --spectrum-infield-button-height: var(--spectrum-component-height-75);
+    --spectrum-infield-button-width: var(--spectrum-component-height-75);
+    --spectrum-infield-button-stacked-fill-padding-inline: var(--spectrum-in-field-button-edge-to-disclosure-icon-stacked-small);
+    --spectrum-infield-button-stacked-fill-padding-outer: var(--spectrum-in-field-button-outer-edge-to-disclosure-icon-stacked-small);
+    --spectrum-infield-button-stacked-fill-padding-inner: var(--spectrum-in-field-button-inner-edge-to-disclosure-icon-stacked-small);
 }
 
-:host([inline='end']) .fill {
-    border-start-start-radius: var(--mod-infield-button-border-radius-reset, var(--spectrum-infield-button-border-radius-reset));
-    border-end-start-radius: var(--mod-infield-button-border-radius-reset, var(--spectrum-infield-button-border-radius-reset));
+:host([size='l']) {
+    --spectrum-infield-button-height: var(--spectrum-component-height-200);
+    --spectrum-infield-button-width: var(--spectrum-component-height-200);
+    --spectrum-infield-button-stacked-fill-padding-inline: var(--spectrum-in-field-button-edge-to-disclosure-icon-stacked-large);
+    --spectrum-infield-button-stacked-fill-padding-outer: var(--spectrum-in-field-button-outer-edge-to-disclosure-icon-stacked-large);
+    --spectrum-infield-button-stacked-fill-padding-inner: var(--spectrum-in-field-button-inner-edge-to-disclosure-icon-stacked-large);
 }
 
-:host([inline='start']) .fill {
-    border-start-end-radius: var(--mod-infield-button-border-radius-reset, var(--spectrum-infield-button-border-radius-reset));
-    border-end-end-radius: var(--mod-infield-button-border-radius-reset, var(--spectrum-infield-button-border-radius-reset));
+:host([size='xl']) {
+    --spectrum-infield-button-height: var(--spectrum-component-height-300);
+    --spectrum-infield-button-width: var(--spectrum-component-height-300);
+    --spectrum-infield-button-stacked-fill-padding-inline: var(--spectrum-in-field-button-edge-to-disclosure-icon-stacked-extra-large);
+    --spectrum-infield-button-stacked-fill-padding-outer: var(--spectrum-in-field-button-outer-edge-to-disclosure-icon-stacked-extra-large);
+    --spectrum-infield-button-stacked-fill-padding-inner: var(--spectrum-in-field-button-inner-edge-to-disclosure-icon-stacked-extra-large);
 }
 
 :host([block='end']),
 :host([block='start']) {
-    --spectrum-infield-button-width: var(--mod-infield-button-width-stacked, var(--spectrum-in-field-button-width-stacked-medium));
+    --mod-infield-button-width: var(--mod-infield-button-width-stacked, var(--spectrum-in-field-button-width-stacked-medium));
```

Comment:
@marissahuysentruyt @castastrophe We have more mod tokens being defined here. do we want to convert back to spectrum to reduce potential headaches for consumers?

Reply:
I'm working on these currently. Thanks for pointing them out!

Reply:
Don't revert these please - look at the stack first: `--mod-infield-button-width` is being mapped to the more specific: `--mod-infield-button-width-stacked`. This is very intentional. Not every mod mapping is a problem here.